.env.example

```diff
@@ -6,7 +6,30 @@ STRIPE_PUBLISHABLE_KEY=pk_test_your_test_key_here
 STRIPE_SECRET_KEY=sk_test_your_test_secret_here
 STRIPE_WEBHOOK_SECRET=whsec_your_webhook_secret_here
 
-# Email Service (Resend - get from https://resend.com/api-keys)
+# Email Service
+# You can pick between SMTP, Resend and Console.
+# If you pick Resend, you will have to configure the API key.
+# If you pick SMTP, you will have to configure the credentials.
+# No further setup is required for console, but it's for development ONLY.
+# Valid options: smtp, resend, console
+MAIL_PROVIDER=
```

Comment:
It's probably better to put resend by default here and comment the rest as people might think they have to fill both.

Reply:
Fixed in [2ff745f](https://github.com/liana-p/indie-game-store/pull/2/commits/2ff745f381ddfbaed1f9c32433f50d862fc9dd25)